integration_test/third_party_apps_test/applications/dcgm/metadata.yaml

```diff
@@ -74,8 +74,7 @@ expected_metrics:
       - *uuid_label
       - *gpu_number_label
     notes:
-      - &dcgm_profiling_note
-        Not available on GPU models P100 and P4.
+      - &dcgm_profiling_note Not available on GPU models P100 and P4.
```

Comment:
Is this change intentional? If not, can we please keep the note on its own line?

Or is this forced by yamlfmt or something?

Reply:
Never mind, this is the result of dumping out the yaml in a Python script. Let's just keep it this way, I guess.